dockerhub/Dockerfile

```diff
@@ -0,0 +1,65 @@
+FROM php:7.3-cli-stretch
+
+RUN apt-get update -y \
+ && apt-get install -y \
+    unzip wget \
+    libfreetype6-dev libjpeg62-turbo-dev libpng-dev libmemcached-dev \
+    zlib1g-dev libicu-dev libpq-dev libtidy-dev libzip-dev \
+    libldap-dev libgmp-dev \
+    libmagickwand-dev  # for the image magick extension (imagick)
+
+RUN docker-php-source extract \
+ && docker-php-ext-install iconv \
+ && docker-php-ext-configure gd --with-freetype-dir=/usr/include/ --with-jpeg-dir=/usr/include/ \
+ && docker-php-ext-install gd \
+ && docker-php-ext-install intl \
+ && docker-php-ext-install zip \
+ && docker-php-ext-install ldap \
+ && docker-php-ext-install gmp \
+ && docker-php-ext-install mysqli \
+ && docker-php-ext-install pgsql \
+ && docker-php-ext-install pdo \
+ && docker-php-ext-install pdo_mysql \
+ && docker-php-ext-install pdo_pgsql \
+ && docker-php-ext-install tidy \
+ && docker-php-ext-install exif \
+ && docker-php-ext-install bcmath \
+ && docker-php-ext-install bz2 \
+ && yes '' | pecl install memcached && docker-php-ext-enable memcached \
+ && yes '' | pecl install redis && docker-php-ext-enable redis \
+ && yes '' | pecl install imagick && docker-php-ext-enable imagick \
+ && yes '' | pecl install apcu && docker-php-ext-enable apcu \
+ && docker-php-source delete
```

Comment:
The sheer number of extensions being applied makes me think this image is intended for running tests against - but the actual database engines are never installed, and there might be other required components missing too.

Is the intention to use this image for running unit tests? If so, there's some more requirements to install. If _not_, installing a lot of these extensions (both `gd` _and_ `imagick`, multiple db drivers, `ldap`, `redis` which isn't even used in our stack) feels like overkill.

Reply:
Originally was intended for doing doing cow releases as well as running tests.   I'd like to limit this Dockerfile to only doing cow releases - is running tests part of the process, or do we rely purely on travis for this?

Reply:
Generally speaking we rely on Travis, yes - if the branches we're targeting are green when we go to release, we should be okay. Travis will also run against every tagged release, so if it exposes a failure in a release candidate we have an opportunity to resolve it before stable.

The only remaining concern to me is that _by default_ Cow will try to trigger unit tests during `cow release x.y.z`. We can (and do) avoid this using `--skip-tests`, so it's not a blocker, but it might be worth officially shifting that responsibility out of Cow longer term.

Reply:
in a happy day scenario, what is the minimal set of cow commands we'd run to do a release? might be worth adding these to the /dockerhub/README.md

Reply:
You can take a look at the [Core release docs](https://docs.silverstripe.org/en/4/contributing/making_a_silverstripe_core_release/) as a reference - or maybe just link to them. I'm conscious that a tool should have documentation for itself and not necessarily rely on an external process doc, but I'm also wary of duplicating content and risking it getting out of sync in the future.